Issue 733/update favicon wrt system theme

rkoleci opened the pull request:
Favicon should update when system theme changes.

Note: not website theme but the computer theme.

Added a listener for this event .


Fix #733

atulbhatt-system32 commented:
@rkoleci I have updated the related issue can you please update your code to that.

rkoleci commented:
> @rkoleci I have updated the related issue can you please update your code to that.

Updated code.

Theme: Light => favicon Dark
Theme Dark => favicon Light
Theme System:
                          Device Light => favicon Dark
                          Device Dark => favicon Light
                          
                          
I believe this is what you meant to say in text.
In the video at the top its actually different. In video, device overrides theme even when theme is set to Light, Dark (not System).

I made it: allow device override only if theme === System, which I believe makes sense.

Pls check attached video. 

Thanks


Uploading 8d089485-fa59-417c-9836-396cba2cdc03.mp4…

atulbhatt-system32 commented:
> rkoleci

video wasn't uploaded.
[rkoleci](https://github.com/rkoleci)

Check this image. The favicon was supposed to be dark.
